src/SimpleSAML/Metadata/Sources/MDQ.php

```diff
@@ -183,7 +183,8 @@ private function getFromCache(string $set, string $entityId): ?array
             ));
         }
 
```

Comment:
Boyscout: above is several exception handling about the file not existing or the read failing. This can be removed since the Symfony FIle class handles that already.